docs/dev/libs/ml/als.md

```diff
@@ -99,6 +114,26 @@ The alternating least squares implementation can be controlled by the following
         </td>
       </tr>
       <tr>
+        <td><strong>ImplicitPrefs</strong></td>
+        <td>
+          <p>
+            Implicit property of the observations, meaning that they do not represent an explicit
+            preference of the user, just the implicit information how many times the user consumed the
+            (Default value: <strong>false</strong>)
+          </p>
+        </td>
+      </tr>
+      <tr>
+        <td><strong>Alpha</strong></td>
+        <td>
+          <p>
+            Weight of the positive implicit observations. Should be non-negative.
+            Only relevant when ImplicitPrefs is set to true.
+            (Default value: <strong>1</strong>)
```

Comment:
I guess it was left as default 1 for no reason, I don't have any motivation for it. I changed it to 40 according to the paper. Thanks for the suggestion!